selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/Workspaces.java

```diff
@@ -218,6 +251,31 @@ public void clickOnWorkspaceAddProjectButton(String workspaceName) {
         .click();
   }
 
+  public void clickOnRamButton() {
+    seleniumWebDriverHelper.waitVisibility(By.xpath("//div[@che-column-title='RAM']/div")).click();
+  }
+
+  public void clickOnProjectsButton() {
+    seleniumWebDriverHelper
+        .waitVisibility(By.xpath("//div[@che-column-title='Projects']/div"))
+        .click();
+  }
+
+  public void clickOnStackButton() {
+    seleniumWebDriverHelper
+        .waitVisibility(By.xpath("//div[@che-column-title='Stack']/div"))
```

Comment:
It seems can be simplified by using `seleniumWebDriverHelper.waitAndClick()` method